source/Politics.cpp

```diff
@@ -99,13 +99,13 @@ void Politics::Offend(const Government *gov, int eventType, int count)
 				provoked.insert(other);
 			}
 		}
-		else if(count && abs(weight) >= .05)
+		else if(abs(weight) >= .05)
```

Comment:
Reverting unneeded changes.
```suggestion
		else if(count && abs(weight) >= .05)
```

Reply:
forgot about this one you're right !